src/game/ui/debug.ts

```diff
@@ -0,0 +1,56 @@
+import { Text, TextClass } from "kontra";
+import { SwarmClass } from "../entities/swarm";
+import { displayMass } from "./util";
+
+type DebugDisplayProps = {
+    trackedSwarms: SwarmClass[],
+    updateDelay: number    
+} & Partial<Parameters<typeof Text>[0]>;
+
+export class DebugDisplayClass extends TextClass {
+
+    private trackedSwarms: SwarmClass[];
+    private updateTimer: number = 0;
```

Comment:
what is this prop for?

Reply:
tracked swarms is an array to just dump some debug information about the swarms on screen while we dont have proper UI for that.

update timer is a throttling parameter to avoid re-rendering the UI 60 times a second as it seems way too busy then.